Description: Prepend "part_" to partmap module names, in line with grub-install
Author: Colin Watson <cjwatson@ubuntu.com>
Origin: upstream, http://bazaar.launchpad.net/~vcs-imports/grub/grub2-bzr/revision/2426
Bug-Debian: http://bugs.debian.org/584426
Forwarded: yes
Last-Update: 2010-06-04

Index: b/util/grub-mkconfig_lib.in
===================================================================
--- a/util/grub-mkconfig_lib.in
+++ b/util/grub-mkconfig_lib.in
@@ -107,7 +107,7 @@
 
   partmap="`${grub_probe} --device ${device} --target=partmap`"
   for module in ${partmap} ; do
-    echo "insmod ${module}"
+    echo "insmod part_${module}"
   done
 
   fs="`${grub_probe} --device ${device} --target=fs`"
